expyfun/stimuli/tests/test_tracker.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
+from expyfun import ExperimentController
+from nose.tools import assert_raises
+import matplotlib.pyplot as plt
```

Comment:
That does not fix it locally. I messed around with ``conf.py`` (following the error message's suggestions) and that did not fix it either.